src/vm/amd64/jithelpers_fast.S

```diff
@@ -163,12 +179,32 @@ LEAF_ENTRY JIT_WriteBarrier, _TEXT
         // Touch the card table entry, if not already dirty.
         shr     rdi, 0Bh
         cmp     byte ptr [rdi + rax], 0FFh
-        // jne     UpdateCardTable
         .byte 0x75, 0x02
+        // jne     UpdateCardTable
         REPRET
 
     UpdateCardTable:
         mov     byte ptr [rdi + rax], 0FFh
+
+#ifdef FEATURE_MANUALLY_MANAGED_CARD_BUNDLES
+        NOP_3_BYTE // padding for alignment of constant
```

Comment:
There is 6 byte nop instruction that would better here - check the Intel manual:

```
66 NOP DWORD ptr [EAX + EAX*1 + 00H]
Byte sequence: 66 0F 1F 44 00 00H
```